app/src/main/res/layout/activity_main.xml

```diff
@@ -39,27 +39,42 @@
         android:layout_marginTop="16dp"
         android:text="@string/feels_like"
         android:textSize="20sp"
+        app:layout_constraintStart_toStartOf="@+id/temperature"
         app:layout_constraintTop_toBottomOf="@+id/temperature"
-        tools:ignore="MissingConstraints"
-        tools:layout_editor_absoluteX="48dp" />
+        tools:ignore="MissingConstraints" />
 
     <ImageView
-        android:layout_width="94dp"
+        android:id="@+id/imageView"
+        android:layout_width="0dp"
         android:layout_height="88dp"
         android:layout_marginStart="120dp"
         android:layout_marginTop="12dp"
+        android:layout_marginEnd="63dp"
         android:src="@drawable/sun"
-        app:layout_constraintStart_toEndOf="@id/temperature"
+        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/city_name" />
 
     <TextView
+        android:id="@+id/info"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginStart="48dp"
         android:layout_marginTop="44dp"
         android:text="@string/info"
         android:textSize="25sp"
+        app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/feels_like"
-        tools:ignore="MissingConstraints"
-        tools:layout_editor_absoluteX="48dp" />
+        tools:ignore="MissingConstraints" />
+
+
+    <Button
```

Comment:
I'm not seeing this button in the MainActivity when I run the app in the emulator.